src/js/pensions/config/form.js

```diff
@@ -783,13 +792,180 @@ const formConfig = {
+                  'ui:options': {
+                    hideIf: (form, index) => _.get(['dependents', index, 'relationship'], form) !== 'child'
+                  },
+                  'ui:title': 'Has your child ever been married?',
+                  'ui:widget': 'yesNo'
+                },
+                married: {
+                  'ui:required': (form, index) => _.get(['dependents', index, 'relationship'], form) === 'child',
+                  'ui:options': {
+                    expandUnder: 'previouslyMarried'
+                  },
+                  'ui:title': 'Are they currently married?',
+                  'ui:widget': 'yesNo'
+                }
+              }
+            }
           }
         },
+        childrenAddress: {
+          path: 'household/dependents/child/address/:index',
+          title: item => `${item.fullName.first} ${item.fullName.last} net worth`,
+          showPagePerItem: true,
+          arrayPath: 'dependents',
+          itemFilter: (item) => !!(item.relationship === 'child'),
+          schema: {
+            type: 'object',
+            properties: {
+              dependents: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  properties: {
+                    childNotInHousehold: dependents.items.properties.childNotInHousehold,
+                    childAddress: dependents.items.properties.childAddress,
+                    personWhoLivesWithChild: dependents.items.properties.personWhoLivesWithChild,
+                    monthlyPayment: dependents.items.properties.monthlyPayment
+                  }
+                }
+              }
+            }
+          },
+          uiSchema: {
+            dependents: {
+              items: {
+                'ui:title': createDisclosureTitle('fullName', 'Address'),
+                childNotInHousehold: {
+                  'ui:title': 'Does your child live with you?',
+                  'ui:required': (form, index) => _.get(['dependents', index, 'relationship'], form) === 'child',
+                  'ui:widget': 'radio',
+                  'ui:options': {
+                    hideIf: (form, index) => _.get(['dependents', index, 'relationship'], form) !== 'child',
+                    labels: {
+                      'false': 'Yes',
+                      'true': 'No'
+                    }
+                  }
+                },
+                childAddress: _.merge(address.uiSchema('Child address', false, (form, index) => !!_.get(['dependents', index, 'childNotInHousehold'], form)),
+                  {
+                    'ui:options': {
+                      expandUnder: 'childNotInHousehold',
+                      hideIf: (form, index) => _.get(['dependents', index, 'relationship'], form) !== 'child' // included to prevent from showing unexpectedly
+                    }
+                  }
+                ),
+                personWhoLivesWithChild: _.merge(fullNameUI,
```

Comment:
It looks like this is always required, and it's blocking you from moving to the next page in the form if you answer Yes to "Does your child live with you?"